homeassistant/components/plugwise/climate.py

```diff
@@ -0,0 +1,250 @@
+"""Plugwise Climate component for HomeAssistant."""
+
+import logging
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
```

Comment:
Please add a blank line between 3rd party and homeassistant import groups.

Reply:
Changed accordingly

Reply:
This still needs updating.

Reply:
This is fixed now, yeah. But I see now that there's an unnecessary blank line below.